ci/jenkins/jobs/projects.yaml

```diff
@@ -1147,3 +1147,59 @@
              - timeout:
                  timeout: 600
                  type: absolute
+      - '{name}-{test_name}-for-pull-request':
+            test_name: e2e-vm
+            node: 'antrea-test-node'
+            description: 'This is the {test_name} test for {name}.'
+            branches:
+                - ${{sha1}}
+            builders:
+                - builder-e2e-vm
+            trigger_phrase: ^(?!Thanks for your PR).*/test-(e2e-vm).*
```

Comment:
better to use 'test-vm-e2e' as the trigger phrase.

Reply:
Sure